Header header
string filter_name
float32 measurement_range
int32 original_points_size
int32 filtered_points_size
int32 original_ring_size
int32 filtered_ring_size
float32 exe_time
